app/views/chats/show.html.erb

```diff
@@ -3,60 +3,80 @@
  data-chat-subscription-chat-id-value="<%= @chat.id %>" 
 data-chat-subscription-current-user-id-value="<%= current_user.id %>"> 
  <!-- Your existing chat area HTML --> </div>
+<style>
+  .gradient-custom {
+/* fallback for old browsers */
+background: #fccb90;
 
+/* Chrome 10-25, Safari 5.1-6 */
+background: -webkit-linear-gradient(to bottom right, rgba(252, 203, 144, 1), rgba(213, 126, 235, 1));
+
+/* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
```

Comment:
If you have no requirement this type of line so Please remove it  because it is taking too much space in our source code so please try to write less of code

Reply:
okk I will remove it